src/Components/test/E2ETest/ServerExecutionTests/ServerAuthTest.cs

```diff
@@ -62,8 +62,8 @@ private void PerformReconnection()
             ((IJavaScriptExecutor)Browser).ExecuteScript("Blazor._internal.forceCloseConnection()");
 
             // Wait until the reconnection dialog has been shown but is now hidden
-            new WebDriverWait(Browser, TimeSpan.FromSeconds(10))
-                .Until(driver => driver.FindElement(By.Id("components-reconnect-modal"))?.GetCssValue("display") == "none");
+            var reconnectModel = Browser.Exists(By.Id("components-reconnect-modal"));
+            Browser.Equal("none", () => reconnectModel.GetCssValue("display"));
```

Comment:
I'm not certain how this is related but totally believe you're doing it for a good reason.

Reply:
I've been changing our tests to use our assertion library. Hopefully we get a better understanding of why the test failed including console log, a screenshot of the page, and the markup.